src/components/WwwFrame/WwwPage.vue

```diff
@@ -1,10 +1,9 @@
 <template>
 	<div class="www-page">
 		<global-promo-contentful v-show="!isKivaAppReferral" />
-		<the-new-header v-if="isNavUpdateExp" />
 		<the-header
-			v-else
 			v-show="!isKivaAppReferral"
+			:is-nav-update-exp="isNavUpdateExp"
```

Comment:
Would it be worth moving the experiment assignment/tracking to the header? It would eliminate one prop drilling.